README.md

```diff
@@ -7,3 +7,67 @@
 PolicyEngine US is a microsimulation model of the US state and federal tax and benefit system.
 
 To install, run `pip install policyengine-us`.
+
+## START
+
+### When starting to work on a specific policy program
+- First check the legal code of the program
+ - Model based on legal code structure
+- Second check newest tax form for inflation adjusted amount confirmation
```

Comment:
Could we add a section such as "Contributing on codespaces"
with the steps from the doc:
https://docs.google.com/document/d/1TTksYmtIXSNOG47H4m3733pvfvf2e3OXOvVQXE92FCQ/edit

Reply:
Also a section on contributing on vscode
1 page here:
https://docs.google.com/document/d/1HD4U0GZqHXIoPtnGMzueCUC-NYPiELN0hlVxmtSSEv8/edit